xtract-core/src/main/scala/com/lucidchart/open/xtract/XPath.scala

```diff
@@ -19,6 +20,11 @@ case class RecursiveXPathNode(key: String) extends XPathNode {
   override def toString = s"//$key"
 }
 
+case class WildCardXPathNode(regex: Regex) extends XPathNode {
+  def apply(xml: NodeSeq): NodeSeq = (xml \\ "_").filter(node => node.label.matches(regex.regex))
```

Comment:
Should this be `xml \ "_"`? or is the intention to search recursively? or maybe we should have both \? and \\? for non-recursive and recursive?

Reply:
Hey, yes makes sense. I will add both